packages/minter-contracts/bin-ts/english_auction_fa2.tz.ts

```diff
@@ -0,0 +1,2 @@
+import { EnglishAuctionFa2CodeType } from './type-aliases';
```

Comment:
This is missing in type-aliases

Reply:
Just noticed this. `@taquito/michel-codec` has trouble parsing some of the contracts, and the recent rebase looks to have made it fail to parse this one as well. 46d87ed will address this.